src/main/webapp/index.html

```diff
@@ -20,6 +20,43 @@
     <title>Mintern</title>
   </head>
   <body>
-      
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <!-- Logo -->
+      <a class="navbar-brand" href="#">
+        <img src="images/mintern-logo.png" height="50" alt="Mintern logo">
+      </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Consider wrapping this line to improve readability.

Reply:
Done!